DigitalLearningSolutions.Web/Views/LearningMenu/Tutorial/ContentViewer.cshtml

```diff
@@ -39,20 +39,22 @@
 <div class="nhsuk-grid-row">
   <div class="nhsuk-grid-column-full">
     <h1 id="page-heading" class="nhsuk-heading-xl">@Model.TutorialName</h1>
-
-    @{ var contentType = "tutorial"; }
-    <partial name="Shared/_JavaScriptDisabledError" model="contentType" />
-
-    <iframe src="@Model.ContentSource" class="nhsuk-grid-column-full nhsuk-u-margin-bottom-6 js-only-block" height="800" id="content-viewer_iframe">
-    </iframe>
-
-    <a class="nhsuk-button js-only-inline"
-       id="content-viewer_enter-fullscreen-button"
-       href="#">
-      Fullscreen
-    </a>
   </div>
 </div>
+  
+@{ var contentType = "tutorial"; }
+<partial name="Shared/_JavaScriptDisabledError" model="contentType" />

```

Comment:
Does it still look okay without js? 🙂

Reply:
The error message is being shown as expected.